frontend/src/pages/wizard.vue

```diff
@@ -78,82 +78,79 @@ meta:
   layout: server
 </route>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
-import { useRoute } from 'vue-router';
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import { useRouter } from 'vue-router';
 import { useI18n } from 'vue-i18n';
 import { getStartupApi } from '@jellyfin/sdk/lib/utils/api/startup-api';
-import { useSnackbar } from '@/composables';
-
-export default defineComponent({
-  setup() {
-    const { t } = useI18n();
-    const route = useRoute();
-
-    route.meta.title = t('wizard.setupWizard');
-
-    return {
-      useSnackbar
-    };
-  },
-  data() {
-    return {
-      wizardStage: 1,
-      maxWizardStage: 1
-    };
-  },
-  computed: {
-    heading(): string {
-      switch (this.wizardStage) {
-        case 1: {
-          return this.$t('wizard.languageLocale');
-        }
-        case 2: {
-          return this.$t('wizard.administratorAccount');
-        }
-        case 3: {
-          return this.$t('wizard.preferredMetadataLanguage');
-        }
-        case 4: {
-          return this.$t('wizard.remoteAccess');
-        }
-      }
-
-      return '';
+import { useRemote, useSnackbar } from '@/composables';
+
+const { t } = useI18n();
+const router = useRouter();
+const remote = useRemote();
+
+const wizardStage = ref(1);
+const maxWizardStage = ref(1);
+
+const heading = computed(() => {
+  switch (wizardStage.value) {
+    case 1: {
+      return t('wizard.languageLocale');
+    }
+    case 2: {
+      return t('wizard.administratorAccount');
     }
-  },
-  methods: {
-    async completeWizard(): Promise<void> {
-      try {
-        const api = this.$remote.sdk.oneTimeSetup(
-          this.$remote.auth.currentServer?.PublicAddress || ''
-        );
-
-        await getStartupApi(api).completeWizard();
-        // Redirect to setup complete page
-        this.$router.replace('/server/login');
-      } catch (error) {
-        console.error(error);
-        this.useSnackbar(this.$t('wizard.completeError'), 'success');
-      }
-    },
-    changeStep({ step }: { step: number }): void {
-      if (step === 4) {
-        this.completeWizard();
-      } else {
-        this.wizardStage += 1;
-      }
-
-      // This allows the return to previous steps, but not going forward past incomplete steps
-      if (this.wizardStage > this.maxWizardStage) {
-        this.maxWizardStage = this.wizardStage;
-      }
-    },
-    previousStep(): void {
-      this.wizardStage -= 1;
+    case 3: {
+      return t('wizard.preferredMetadataLanguage');
+    }
+    case 4: {
+      return t('wizard.remoteAccess');
     }
   }
+
+  return '';
 });
+
+/**
+ * Completes server setup
+ */
+async function completeWizard(): Promise<void> {
+  try {
+    const api = remote.sdk.oneTimeSetup(
+      remote.auth.currentServer?.PublicAddress || ''
+    );
+
+    await getStartupApi(api).completeWizard();
+    // Redirect to setup complete page
+    router.push('/server/login');
```

Comment:
```suggestion
    router.replace('/server/login');
```

So user can't go back